### ISSUE TITLE
does not work with 3.32.2

### ISSUE THREAD

**Issue by jarrard** [The-Panacea-Projects/Gnomenu#125]
Pop_OS Gnome 3.32.2 here and this menu doesn't work. :(

**Comment by The-Panacea-Projects** [comment]
Have you installed Dependency "gnome-menus"

**Comment by jarrard** [comment]
yes, it comes with pop os 19.04 it seems.

**Comment by The-Panacea-Projects** [comment]
Did you follow the instruction here https://github.com/The-Panacea-Projects/Gnomenu/issues/118#event-2487137706

**Comment by jarrard** [comment]
Yeah I downloaded the master and put it over top of the one from the website, also tried it in a clean folder. Is there a away I can get a log output for this extension? 

PS. I'm using Plasma5 now as it works better with games somehow, but can still login to gnome and test etc...

**Comment by passingthru67** [comment]
You can use the journalctl --since and --until options like below ..
journalctl --since "2019-07-30 08:15:00" --until "2019-07-30 08:17:00"

If you want to send the logs to a file, do this ..
journalctl --since "2019-07-30 08:15:00" --until "2019-07-30 08:17:00" > journal.txt

Don’t forget to replace the date/time with the date & time of the issue.

**Comment by methilnet** [comment]
With Pop_OS 19.04 you need package gir1.2-gmenu-3.0 and it dependency libgnome-menu-3-0 for this extension to work. After that "Download button" top right. Rename the extracted folder to " gnomenu@panacier.gmail.com " and move to your extensions folder. retart gnome-shell or logout/login.

For the record the error message was:
Aug 10 17:25:33 pop-os gnome-shell[3231]: Extension "gnomenu@panacier.gmail.com" had error: Error: Requiring GMenu, version none: Typelib file for namespace 'GMenu' (any version) not found

**Comment by jarrard** [comment]
ok cheers, didn't know that. Might mentioned it in the extensions comment section once I test it out.

**Comment by LordPhoenixfr** [comment]
Got same problem here on Ubuntu 19.04 and same solution was successful…

**Comment by IncognitaDev** [comment]
I do all steps, but I got this error: 
No JS module 'tweener' found in search path

**Comment by POpus** [comment]
> I do all steps, but I got this error:
> No JS module 'tweener' found in search path

The solution to that is possibly here by modifying the path to tweener in the 'extension.js' file => https://gitlab.com/jenslody/gnome-shell-extension-panel-osd/-/merge_requests/30/diffs

But it's not there in my PopOS installation but in 'myWorkspaceThumbnail.js'. Made the same change and I got another error

**Comment by IncognitaDev** [comment]
> > I do all steps, but I got this error:
> > No JS module 'tweener' found in search path
> 
> The solution to that is possibly here by modifying the path to tweener in the 'extension.js' file => https://gitlab.com/jenslody/gnome-shell-extension-panel-osd/-/merge_requests/30/diffs
> 
> But it's not there in my PopOS installation but in 'myWorkspaceThumbnail.js'. Made the same change and I got another error

I change distro and dont remind to exclude my comment, but tank you, will help someone.
